src/DataCollector/GuzzleCollector.php

```diff
@@ -41,17 +41,34 @@ class GuzzleCollector extends DataCollector
     public function __construct($maxBodySize = self::MAX_BODY_SIZE)
     {
         $this->maxBodySize = $maxBodySize;
-        $this->history = new \SplObjectStorage();
+        $this->history = new \ArrayObject();
```

Comment:
Array here, and change the `getHistory()` method to return a reference to the array.